packages/client/src/client.ts

```diff
@@ -96,6 +97,7 @@ export const buildClient = <Plugins extends Record<string, XataPlugin> = {}>(plu
       const apiKey = options?.apiKey || getAPIKey();
       const cache = options?.cache ?? new SimpleCache({ defaultQueryTTL: 0 });
       const trace = options?.trace ?? defaultTrace;
+      const clientName = options?.clientName ?? 'sdk';
```

Comment:
@SferaDev I don't think we want this fallback to `'sdk'`? We could leave it `undefined` or set it to `unknown`. 

But clientName is actually the 'service' e.g. `'frontend' | 'cli' | 'playground'` so saying 'sdk' doesn't make sense here.

Reply:
I wanted to have it so that in mixpanel we can differentiate from raw API requests done maybe with curl, insomnia, postman and the ones coming from an SDK client. Does that make sense?

Reply:
Yes we definitely want that, and we have it via: `X-Xata-Agent=client=TS_SDK;`

https://github.com/xataio/client-ts/pull/768/files#diff-76651c34752b02dfceb46181b8b38ef33ea21122080a391c59b93a19647f178fR139

BTW: This is why I preferred the name `service` -> `clientName`

Reply:
Okay reverting then, will make it nullable though instead of undefined